guacamole-common-js/src/main/webapp/modules/OnScreenKeyboard.js

```diff
@@ -455,6 +455,17 @@ Guacamole.OnScreenKeyboard = function(layout) {
 
     };
 
+    /**
+     * Resets the state of this keyboard, releasing all keys, and firing keyup
+     * events for each released key.
+     */
+    this.reset = function() {
```

Comment:
Though historical code has not followed this, please define a name for the function:

    this.reset = function reset() {
        ...
    };

This is being done for all new code as doing otherwise results in unreadable stack traces.